contents/docs/web-analytics/revenue-tracking.mdx

```diff
@@ -0,0 +1,146 @@
+
+Once you've connected your Stripe account, you'll be able to choose the `stripe_price` table and the `amount`, `currency` and `created_at` columns from the Stripe data.
+
+And that's all, you'll immediately see your revenue data in your Web Analytics dashboard.
+
+#### What if you don't support the currency on my revenue event?
+
+We'll track that revenue event as if it had 0 value. You can always open a feature request to have us include a new currency to our list.
+We're always monitoring new currencies and adding them as we see fit.
+
+## Using revenue data beyond web analytics
+
+### Using revenue data in product analytics
+
+You can absolutely use your revenue data for deeper Product Analytics beyond the Web Analytics dashboard. This allows you to:
+
+- Create custom revenue funnels
+- Analyze user journeys that lead to purchases
+- Segment users by revenue contribution
+
+When working with revenue data in Product Analytics, you may want to use the [`convertCurrency` function](#querying-converted-rates) for accurate reporting across different currencies.
+
+### Sending backend events for revenue tracking
+
+You can send revenue tracking events from your backend systems, which is especially useful for tracking purchases that complete outside of the browser.
+
+When sending backend events, you must include a `session_id` to ensure they appear in Web Analytics. The session ID connects the backend purchase event with the user's web session.
+
+The `session_id` should be a UUID V7 that uniquely identifies the user's session.
+You can either generate a random one, but ideally you should reuse the same value we generated in our Frontend SDK so that all events belong to the same session in our UI.
+You can learn more about how to do this in the [session docs](/docs/data/sessions#server-sdks-and-sessions).
+
+Example backend event:
+```javascript
+posthog.capture({
+    distinct_id: posthog_distinct_id,
+    event: 'Purchase Succeeded',
+    properties: {
+        '$session_id': posthog_session_id
+    }
+})
+```
+
+## Troubleshooting & FAQ
+
+### Can I change the default reporting currency in PostHog?
+
+Yes. You can select your desired currency in the **Revenue** tab in data management. This is the currency we'll **report** your revenue in our web analytics dashboard.
+
+### How do you convert renvenue in different currencies?
+
+We automatically convert revenue from other currencies into your chosen reporting currency – i.e. if someone makes a purchase in Canadian Dollars (CAD) and your reporting currency is US Dollars (USD), it's converted into USD in PostHog to ensure consistency.
+
+PostHog's currency conversion data is sourced from [Open Exchange Rates](https://openexchangerates.org/). The exchange rate data is updated at hourly and stored with daily granularity.
+
+This ensures that your revenue data is converted using recent exchange rates, providing reliable revenue analytics across different currencies.
```

Comment:
We should clarify that we use "the exchange rate from the time of the event" rather than "the most recent exchange rate"